### ISSUE TITLE
pdf-parser.lisp will be upgraded to parse pdf version 1.7 or later?

### ISSUE THREAD

**Issue by anranyicheng** [mbattyani/cl-pdf#31]
hello, We know that the version of most pdf files now is 1.7, so will this parsed file([pdf-parser.lisp](https://github.com/mbattyani/cl-pdf/blob/master/pdf-parser.lisp)) be updated?

**Comment by hectormonacci** [comment]
Maybe some bounty could be offered for such an improvement. I would be glad to offer!

**Comment by adlai** [comment]
@anranyicheng Why did you close the issue? If anything, this one might actually have useful result...

**Comment by anranyicheng** [comment]
> @anranyicheng Why did you close the issue? If anything, this one might actually have useful result...

new library base on cl-pdf, https://github.com/splittist/pdfreader

**Comment by adlai** [comment]
> new library base on cl-pdf, https://github.com/splittist/pdfreader

That library does not depend on this one.

**Comment by anranyicheng** [comment]
Yes, my company (that is, me personally) is not profitable yet, maybe next year, it will be profitable, and I am willing to provide funds
